docs/reference/docs/delete-by-query.asciidoc

```diff
@@ -463,6 +472,11 @@ Which results in a sensible `total` like this one:
 ----------------------------------------------------------------
 // TESTRESPONSE
 
+Setting `slices` to `auto` will let Elasticsearch choose the number of slices
+to use. This setting will use one slice per shard, up to a certain limit. If
+there are multiple source indices, it will choose the number of slices based
```

Comment:
Sounds good, I'll open an issue for it after this is merged. I haven't looked at how search requests with multiple indices work so I'm not familiar with what would change in that case